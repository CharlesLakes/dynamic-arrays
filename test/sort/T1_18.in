10
48 92 55 86 46 49 70 73 11 86