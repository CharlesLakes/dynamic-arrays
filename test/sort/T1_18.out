11 46 48 49 55 70 73 86 86 92